samples/interactive-tutorials/src/main/java/events/WriteUserEvent.java

```diff
@@ -38,11 +38,11 @@ public class WriteUserEvent {
 
   public static void main(String[] args)
       throws IOException, ExecutionException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample.
-    String projectId = ServiceOptions.getDefaultProjectId();
+    // TODO: Set projectId to your Google Cloud Platform project ID.
+    String projectId = "my-project";
     String defaultCatalog =
         String.format("projects/%s/locations/global/catalogs/default_catalog", projectId);
-    // visitorId generated randomly.
+
     String visitorId = UUID.randomUUID().toString();
```

Comment:
Do we want to add a line comment why are we using a random visitorId, or some guidance on what this value should be replaced with while writing user event?